src/main/java/com/codegym/jrugotom5/repository/AdvertRepository.java

```diff
@@ -8,4 +8,5 @@
 
 public interface AdvertRepository extends CrudRepository<Advert, Long> {
     List<Advert> findAllByCreatedDateBetween(LocalDate from, LocalDate to);
+    Advert findAdvertById(Long id);
```

Comment:
I think it is safer to return Optional<Advert> here and then `findAdvertById().orElseThrow(*your exception*)` in service.

Reply:
This method is already in CrudRepository, so I deleted it